exercises/parallel-letter-frequency/src/test/java/ParallelLetterFrequencyTest.java

```diff
@@ -151,13 +152,13 @@ public void allThreeAnthemsTogetherProduceCorrectCounts() {
         assertEquals(new Integer(2), p.letterCounts().get((int) 'ü'));
     }
 
+    @Ignore("Remove to run test")
     @Test
-    @Ignore
     public void tenThreadPoolIsFasterThanSingleThreadPool()
```

Comment:
This test failed when Travis ran it which failed the build :(

Reply:
Thanks @FridaTveit I see, I wonder if that is an artifact of the CI environment..I am starting to feel like this test is too fragile.  Let me work further on it.